src/main/java/org/sagebionetworks/web/client/widget/discussion/DiscussionThreadWidget.java

```diff
@@ -34,7 +39,28 @@ public void configure(DiscussionThreadBundle bundle) {
 		view.setActiveUsers(bundle.getActiveAuthors().toString());
 		view.setNumberOfReplies(bundle.getNumberOfReplies().toString());
 		view.setNumberOfViews(bundle.getNumberOfViews().toString());
-		view.setLastActivity(bundle.getLastActivity().toString());
+		view.setLastActivity(gwtWrapper.getFormattedDateString(bundle.getLastActivity()));
+		view.setAuthor(bundle.getCreatedBy());
+		view.setCreatedOn(gwtWrapper.getFormattedDateString(bundle.getCreatedOn()));
+		if (bundle.getNumberOfReplies() > 0) {
```

Comment:
I think the click handler to show replies should always be set in the view.  And if 0 replies, then simply don't show the reply count.